src/components/NamespaceList/NamespaceListRow.tsx

```diff
@@ -48,7 +48,12 @@ const NamespaceListRow: React.FC<React.PropsWithChildren<RowFunctionArgs<Namespa
   return (
     <>
       <TableData className={namespaceTableColumnClasses.name} data-test="app-row-test-id">
-        {obj.metadata.name}
+        <Link
+          to={APPLICATION_LIST_PATH.createPath({ workspaceName: obj.metadata.name })}
+          title="Go to this namespace"
+        >
+          {obj.metadata.name}
+        </Link>
```

Comment:
We already have a button on the namespace list page, `Go to this namespace`. Do we want to remove that?
In the UI, Any hyperlink on the name takes the user to the details page of that resource and since we dont have a details page for the namespace Ux suggested the addition of the button as an action that takes user to the application list page.

Reply:
@brianwcook This design was a bit tricky. Linking a namespace given an impressions that it will take user to the details view of the namespace but that's not the case. The goal of the button was to let user set the context by selecting an action on the namespace. My original design had that action on the kebab menu but the feedback I received was that it would a bit hidden and it would helpful to have it more visible in the table. The result is the current design that you are looking at. I agree, it looks a bit clunky but my understanding was that it was a stopgap solution until we are able to support the creation of namespace within Konflux.